app/src/services/attestation.tsx

```diff
@@ -413,6 +308,14 @@ export const AttestationProvider: React.FC<AttestationProviderParams> = ({ child
         .subscribe(handleOfferWithParam)
       setOfferSubscription(offerSub)
     }
+
+    while (true) {
```

Comment:
Can we get stuck here forever?

Reply:
it won't get stuck unless someone awaits the start method, but I see what you mean, I'll clean it up